lib/user_agent/browsers/spreaker.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+class UserAgent
+  module Browsers
+    # Spreaker for Android 4.17.0 release:2021-05-05 device:phone (samsung SM-G965U; Android 10; en_US)
+    # Spreaker Android
+    # Spreaker 7.26.1 rv:1337 (iPhone10,3; iOS 14.4.2; en_US)
+    # Spreaker 6.8.0 rv:1168 (iPad2,4; iPhone OS 9.3.5; en_GB) SPKAudioPlayer
+    # Spreaker 7.9.1 rv:1263 (iPod7,1; iOS 11.0.3; en_US) SPKAudioPlayer
+    # Spreaker File Downloader
+    class Spreaker < Base
+      IPHONE_OS     = 'iPhone OS'
+      SPREAKER      = 'Spreaker'
+      VERSION_REGEX = /Spreaker (for Android )?(?<version>[\d\.]+)/.freeze
```

Comment:
```suggestion
      VERSION_REGEX = /\ASpreaker (?:for Android )?(?<version>[\d\.]+)/.freeze
```

(this makes the `for Android` group non-capturing and performs better)